app/assets/stylesheets/style_updates.css.scss

```diff
@@ -60,21 +61,18 @@ html {
   height:100%;
   top:50px;
   left:0px;
-  font-family: 'Tangerine', cursive;
   .container {
     padding-left: 28vw;
     padding-top: 10vw;
-  }
-
-  .form {
-    max-width: 200px;
-
-    .label{
-      font-size: 35px;
-      color: #000000;
+      .form {
+        max-width: 200px;
+        .label{
+          font-size: 35px;
+          color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.